app/main.py

```diff
@@ -36,12 +41,19 @@ def knowledge_panel(
             data_quality_kp(facet=facet_name, value=facet_value, country=country)
         )
     except Exception as Argument:
```

Comment:
Don't capitalize a variable. Argument is a variable. Also if you don't use it underneath, do not use it:

```suggestion
    except Exception:
```

You should correct it everywhere it's the case.